frontend/src/component/project/Project/Project.tsx

```diff
@@ -160,6 +154,13 @@ export const Project = () => {
             path: `${basePath}/change-requests`,
             name: 'change-request',
             isEnterprise: true,
+            label: simplifyProjectOverview
+                ? () => (
+                      <ActionableChangeRequestsIndicator>
+                          Change requests
```

Comment:
Why are we passing this in? Are we using any other texts?

Reply:
Good point. Yeah, We can remove that.